bin_tests/src/lib.rs

```diff
@@ -69,6 +69,11 @@ fn inner_build_artifact(c: &ArtifactsBuild) -> anyhow::Result<PathBuf> {
     /// it's directory
     static ARTIFACT_DIR: OnceCell<PathBuf> = OnceCell::new();
     let artifact_dir = ARTIFACT_DIR.get_or_init(|| {
+        // If the CARGO_TARGET_DIR env var is set, then just use that.
+        if let std::result::Result::Ok(env_target_dir) = env::var("CARGO_TARGET_DIR") {
```

Comment:
```suggestion
        if let Ok(env_target_dir) = env::var("CARGO_TARGET_DIR") {
```

Reply:
I had to provide the path because [anyhow::Ok is included](https://github.com/DataDog/libdatadog/blob/014b27e731d42445916090b501190fbff0a11de2/bin_tests/src/lib.rs#L8). But removing that `use` doesn't break anything so I'll do that.